### PR TITLE
Scaffold project upgrades page

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

Scaffold the upgrades tab on the settings page.

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2160** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2160" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2160?utm_source=stack-comment).

**Review by wraeth-eth — APPROVED** [review]

**Comment by tomquirk** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2160?utm_source=stack-comment).


Job Reference: 2UzpC3JR8gjxEQLDrz47

**Pull request merged**
